CONTRIBUTING.md

```diff
@@ -10,4 +10,6 @@ For other changes, please raise an issue first to check that the change aligns w
 
 If your pull request or proposed change is for a bug that is causing problems for you, please also [contact support](http://octopusdeploy.com/support) referencing your pull request to make sure it gets prioritized properly. They may also be able to find a work around in the meantime.
 
-If you have an idea or a feature request, please post it to [our UserVoice site](http://octopusdeploy.uservoice.com) so others can vote for it. 
+If you have an idea or a feature request, please post it to [our UserVoice site](http://octopusdeploy.uservoice.com) so others can vote for it.
+
+If you're a keen contributor, a regular OctopusDSC user or just curious, please feel free to join us in the `#octopusdsc` channel at our [Community Slack](https://octopususergroup.slack.com/)
```

Comment:
You'll need to use the special magic invite link here, otherwise they wont be able to join